sonar-project.properties

```diff
@@ -2,7 +2,8 @@ sonar.projectKey=ipv-core-front
 sonar.organization=govuk-one-login
 
 sonar.sources=src/
-sonar.exclusions=**/assets/**,**/*.test.js,**/*.test.ts,**/service-unavailable-s3/**,**/config/**,**/lib/**,**/*router.js,**/axiosHelper.js,**/app.ts,**/development/**
+# Remove tests, config, routing and development-only code from code coverage
+sonar.exclusions=**/*.test.ts,**/config/**,**/*router.ts,**/app.ts,**/development/**
```

Comment:
Previously we were excluding `lib` which doesn't feel right - this might introduce some uncovered code, but it's probably accurate.

Reply:
It also hides these files from Sonar analysis, which isn't really desirable!